### PR TITLE
Add Weclapp CRM

### PR DESCRIPTION
**Pull request by AWqxKAWERbXo**
A German CRM/ERP-System.

### CODE REVIEW

**Comment by AWqxKAWERbXo** [comment]
not sure about the URL - weclapp provides subdomains for every client, like iamaclientof.weclapp.com

File: _data/other.yml [hunk 1]
```diff
@@ -567,6 +567,14 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://*.weclapp.com/
```
Comment (kmpoppe) [inline _data/other.yml]: To make the PR pass travis, please remove the *.
```suggestion
      url: https://weclapp.com/
```

**Review by kmpoppe — DISMISSED** [review]
Alexa-Rank 138,464.
@alexplusde I took myself the liberty to tweak your PR (image and removing superfluous tags).
Thank you for your contribution!

File: _data/other.yml [hunk 2]
```diff
@@ -567,6 +567,13 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://weclapp.com/
+      img: weclapp.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.weclapp.com/de/blog/2-faktor-authentifizierung/
```
Comment (Carlgo11) [inline _data/other.yml]: The blog post describes U2F implementation which we categorize as hardware 2FA.
```suggestion
      hardware: Yes
      doc: https://www.weclapp.com/de/blog/2-faktor-authentifizierung/
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by AWqxKAWERbXo** [comment]
@kmpoppe Thank you so much!

I somehow clicked the wrong button and dismissed the change at first. It should be ok now, shouldn't it?

File: _data/other.yml [hunk 3]
```diff
@@ -567,6 +567,15 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://weclapp.com/
+      img: weclapp.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+      hardware: Yes
```
Comment (kmpoppe) [inline _data/other.yml]:
```suggestion
```
I now see hardware twice, that should not be the case, I think.

**Comment by ertanoe** [comment]
@kmpoppe Could you please adjust the doc link to be this one here: https://doc.weclapp.com/knowledgebase/zwei-wege-authentifizierung-2fa/
We from weclapp did review this as well and the rest seems to be fine

File: _data/other.yml [hunk 4]
```diff
@@ -567,6 +567,14 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://weclapp.com/
+      img: weclapp.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+      doc: https://www.weclapp.com/de/blog/2-faktor-authentifizierung/
```
Comment (kmpoppe) [inline _data/other.yml]:
```suggestion
      doc: https://doc.weclapp.com/knowledgebase/zwei-wege-authentifizierung-2fa/
```

Change to documentation link as per @ertanoe 's request.

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Sorry about the delay. Your commits have been merged now @alexplusde

**Pull request merged**